_example/as_sysdba/main.go

```diff
@@ -8,14 +8,18 @@ import (
 	_ "github.com/mattn/go-oci8"
 )
 
-func getDSN() string {
-	// same as "sqlplus sys/syspwd@tnsentry as sysdba"
-	return "sys/syspassword@mytnsentry?as=sysdba"
-}
 func main() {
+	if len(os.Args) != 2 {
+		fmt.Printf("ERROR: Please provide a DSN string in ONE argument:\n\n")
+		fmt.Println("Shell-Conversion into DSN string:")
+		fmt.Println("  sqlplus sys/password@tnsentry as sysdba   =>   sys/password@tnsentry?as=sysdba")
+		fmt.Println("  sqlplus / as sysdba                       =>   sys/.@?as=sysdba")
+		fmt.Println("instead of the tnsentry, you can also use the hostname of the IP.")
```

Comment:
What about Easy Connect syntax, which is the preferred first choice instead of bothering with tnsnames.ora files?

Reply:
Yes, I just reused the text `tnsentry` from the example. Easy connect also works. Maybe we should just create a Wiki page for different connect patterns and state the URL in the example.